cedar-policy-core/src/est/annotation.rs

```diff
@@ -0,0 +1,86 @@
+/*
+ * Copyright Cedar Contributors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+use std::collections::BTreeMap;
+
+use serde::{Deserialize, Serialize};
+
+use crate::ast::{self, Annotation, AnyId};
+#[cfg(feature = "wasm")]
+extern crate tsify;
+
+/// Similar to [`ast::Annotations`] but allow annotation value to be `null`
+#[derive(Serialize, Deserialize, Clone, Hash, Eq, PartialEq, PartialOrd, Ord, Debug)]
+#[cfg_attr(feature = "wasm", derive(tsify::Tsify))]
+pub struct Annotations(
+    #[serde(default)]
+    #[serde(skip_serializing_if = "BTreeMap::is_empty")]
+    #[serde(with = "::serde_with::rust::maps_duplicate_key_is_error")]
+    #[cfg_attr(feature = "wasm", tsify(type = "Record<string, Annotation>"))]
```

Comment:
maybe my comment above about `loc`s in the EST should have been here.  I didn't realize there were two `Annotations` types now